### PR TITLE
Create ChiaCryptoMining_WindowsEvent

### PR DESCRIPTION
**Pull request by yaelrbergman**
Required items, please complete
   
   Change(s):
Added ChiaCryptoMining detection for WindowsEvent 
   
Reason for Change(s):
Adding WindowsEvent as a source

   Version Updated:
Yes
   Testing Completed:
Yes

   Checked that the validations are passing and have addressed any issues that are present:
Yes

### CODE REVIEW

File: Detections/WindowsEvent/ChiaCryptoMining_WindowsEvent.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+id: 4d173248-439b-4741-8b37-f63ad0c896ae
```
Comment (oshezaf) [inline Detections/WindowsEvent/ChiaCryptoMining_WindowsEvent.yaml:1]: We use tags to refer to the original for code maintenance reasons. See [here](https://github.com/Azure/Azure-Sentinel/blob/master/Detections/ASimWebSession/PossibleDGAContacts.yaml)

File: Detections/WindowsEvent/ChiaCryptoMining_WindowsEvent.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+triggerThreshold: 0 
+tactics: 
+  - Impact
+relevantTechniques:
+  - T1496
+tags:
+  - Chia 
+query:  |  
+  let iocs = externaldata(DateAdded:string,IoC:string,Type:string,TLP:string) [@"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Sample%20Data/Feeds/ChiaCryptoIOC.csv"] with (format="csv", ignoreFirstRecord=True);
+  let process = (iocs | where Type =~ "process" | project IoC);
```
Comment (oshezaf) [inline Detections/WindowsEvent/ChiaCryptoMining_WindowsEvent.yaml:22]: No need to keep the lists you don't use.

**Review by oshezaf — CHANGES_REQUESTED** [review]

**Review by aprakash13 — COMMENTED** [review]
@yaelrbergman Just wondering if we want a new query for this or do we want to include this in the existing ChiaCryptoMining detection.

**Comment by oshezaf** [comment]
> @yaelrbergman Just wondering if we want a new query for this or do we want to include this in the existing ChiaCryptoMining detection.

@aprakash13 : as noted in another PR - Yael added a new version when the original was too long to add to.

**Review by oshezaf — APPROVED** [review]

**Pull request merged**